LICENSE

```diff
@@ -0,0 +1,5 @@
+Apache License
```

Comment:
Did we check with anyone before making this change?

Reply:
Yes I put this point in our Monday syncup and we agreed on Apache2.0.